src/main/scala/org/clulab/wm/eidos/extraction/Finder.scala

```diff
@@ -7,7 +7,8 @@ import org.clulab.processors.Document
 import org.clulab.wm.eidos.context.{GeoNormFinder, TimeNormFinder}
 
 trait Finder {
-  def extract(doc: Document, initialState: State = new State()): Seq[Mention]
+  // This last argument is optional because not all finders need it and that way a simple default value can be provided.
```

Comment:
This comment is presumably no longer relevant?

Reply:
Indeed, but it was so very insightful that I unconsciously could not part with it.  Gone.